projects/helper/coreAssets.json

```diff
@@ -1950,7 +1950,7 @@
     "XRP": "XRP"
   },
   "polynomial": {
-    "USDC": "0x17C9D8Cef7Ef072844EEaEdE1F9f54C7E3fa8743",
+    "USDC": "0x2369EB4a76d80fBeAa7Aa73e1e1f9EAeE88C07F4",
```

Comment:
fxUSDC is more correct because it represents the amount of usdc staked on the liquidity app, and the difference is calculated on Trade so it gets added twice. Also can you check why doesn't usd0 get callculated, that should be usd0++ token from the usual. This function doesn't add it to tvl idk why